odoo/addons/test_lint/tests/test_manifests.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.modules import get_modules
+from odoo.modules.module import load_manifest, _DEFAULT_MANIFEST
+from odoo.tests import BaseCase
+
+class ManifestLinter(BaseCase):
+
+    def test_manifests_keys(self):
+        known_keys = {
+            *_DEFAULT_MANIFEST, 'name', 'icon', 'addons_path', 'license',
+            'contributors', 'maintainer', 'url', 'web.assets_backend',
```

Comment:
`web.assets_backend` seems to be a mistake introduced in odoo/odoo#69118.

Unless @tde-banana-odoo or @mart-e can explain why it's needed, this should be removed and the keys can be dropped in the *plugin module manifests.

For their pseudo qweb XML files to be detected by the translation export system, all that's needed is to put them under `/static/src/xml`: https://github.com/odoo/odoo/blob/master/odoo/tools/translate.py#L1110-L1113